cmake/CompilerFlagsHelpers.cmake

```diff
@@ -97,7 +99,9 @@ foreach(COMPILER_LANGUAGE ${SUPPORTED_COMPILER_LANGUAGE_LIST})
     set(CMAKE_${COMPILER_LANGUAGE}_STACK_PROTECTION "")
     set(CMAKE_${COMPILER_LANGUAGE}_POSITION_INDEPENDENT "-fPIC")
     set(CMAKE_${COMPILER_LANGUAGE}_VECTORIZE "")
+    set(CMAKE_${COMPILER_LANGUAGE}_IGNORE_WARNINGS "-Wno-deprecated-register -Wno-writable-strings")
```

Comment:
this `else` block is generic where we don't know compiler being used (e.g. Fujitsu). In this case we shouldn't add this flag.